test/unit/test_sap_adt_api_aunit.py

```
@@ -153,6 +165,10 @@ def test_function_group(self):
         xml = Marshal().serialize(build_test_run([('zfugr', 'function-group')]))
         self.assertEqual(xml, EXPECTED_XML_FUGR)
 
+    def test_function_group(self):
```

Flake8 (7.3.0)

[error] redefinition of unused 'test_function_group' from line 164

(F811)